### DISCUSSION TITLE
Global CSS cannot be imported from files other than your Custom <App>.

### DISCUSSION THREAD

**Discussion by gabster-code** [vercel/next.js#40194] [Help]
Guys, what did I do wrong?

I just created a NEW APP via npx create-next-app@latest and NO CHANGES DONE yet but I'm having this issue:
![image](https://user-images.githubusercontent.com/88597385/188275329-dd04cbee-db58-4a43-b66c-6fb25db16e84.png)

Did something change? Earlier today(bout 8hours ago) I also did a new project to practice but it was working that time(Now it also throws the same error as above when running npm run dev)

Please help me.  

`npx --no-install next info`   output:
`

> Operating System:
>       Platform: win32
>       Arch: x64
>       Version: Windows 10 Pro
>     Binaries:
>       Node: 16.17.0
>       npm: N/A
>       Yarn: N/A
>       pnpm: N/A
>     Relevant packages:
>       next: 12.2.6-canary.8
>       eslint-config-next: 12.2.5
>       react: 18.2.0
>       react-dom: 18.2.0
> `

UPDATE: 
I found a template I pushed to Vercel earlier(which is ironically just the base template) and cloned my repo for that.
SAME ERROR OCCURED

Link to vercel site:
https://vercel.com/gabster-code/nextjs/GfxUABTnF2S9hK9XYXsfzwx4cApg

Link to repo I cloned: 
https://github.com/gabster-code/nextjs/tree/main

![image](https://user-images.githubusercontent.com/88597385/188277716-86062779-0cdc-4ae4-8a48-74b1cab3a63f.png)

**Comment by gabster-code** [comment]
Anyone who can help please?

**Comment by icyJoseph** [comment]
Did you figure out what was going on? I cannot find the repository you link to. Got another we could take a look at? 

A bit suspicious that the paths to the `_app.js` using `../../../` as much as they do, are you running the project from the correct location? Interesting case.
